apps/constants/base.py

```diff
@@ -5,20 +5,19 @@
 
 
 # Add-on and File statuses.
-STATUS_NULL = 0
-STATUS_UNREVIEWED = 1
-STATUS_PENDING = 2
-STATUS_NOMINATED = 3
-STATUS_PUBLIC = 4
-STATUS_DISABLED = 5
+STATUS_NULL = 0  # No review type chosen yet, add-on is incomplete.
+STATUS_UNREVIEWED = 1  # Waiting for prelim review.
+STATUS_PENDING = 2  # Personas (lightweight themes) waiting for review.
+STATUS_NOMINATED = 3  # Waiting for full review.
+STATUS_PUBLIC = 4  # Fully reviewed.
+STATUS_DISABLED = 5  # Disabled by mozilla (rejected).
```

Comment:
Rejected, in the case of individual files. Admin disabled in the case of add-on listings. `s/mozilla/Mozilla/`